charts/zabbix/templates/_helpers.tpl

```diff
@@ -35,17 +35,21 @@ Create chart name and version as used by the chart label.
 Common labels
 */}}
 {{- define "zabbix.labels" -}}
-app.kubernetes.io/name: {{ include "zabbix.name" . }}
 helm.sh/chart: {{ include "zabbix.chart" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
+{{ include "zabbix.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
```

Comment:
Hi @fibbs 

Here, can we change  ``app.kubernetes.io/managed-by: {{ .Release.Service }}``by ``app.kubernetes.io/managed-by: Helm``?

Reply:
Hey @aeciopires.
I was thinking the same, but then I compared what helm does it you do a fresh `helm install` and it's using `{{ .Release.Service }}` which is part of the Helm-internal `.Release` object and resolves to "Helm" [according to the official documentation](https://helm.sh/docs/chart_template_guide/builtin_objects/). This makes me believe that this is the "more correct" option

Reply:
Thanks for sharing the documentation. I now understood better.
I agree with you.